core/storage/user/gae_models_test.py

```diff
@@ -31,39 +31,102 @@ class UserSettingsModelTest(test_utils.GenericTestBase):
+    user3_email = 'user3@example.com'
+    user3_role = feconf.ROLE_ID_ADMIN
+    user3_id = 3
+    generic_username = 'user'
+    generic_date = datetime.datetime(2019, 5, 20)
+    generic_image_url = 'www.example.com/example.png'
+    generic_user_bio = 'I am a user of Oppia!'
+    generic_subject_interests = ['Math', 'Science']
+    generic_language_codes = ['en', 'es']
 
     def setUp(self):
         super(UserSettingsModelTest, self).setUp()
         user_models.UserSettingsModel(
             email=self.user_email, role=self.user_role).put()
         user_models.UserSettingsModel(
             email=self.user2_email, role=self.user2_role).put()
+        user_models.UserSettingsModel(
+            email=self.user3_email, role=self.user3_role).put()
+        user_models.UserSettingsModel(
+            id=self.user3_id,
+            email=self.user3_email,
+            role=self.user3_role,
+            username=self.generic_username,
+            normalized_username=self.generic_username,
+            last_agreed_to_terms=self.generic_date,
+            last_started_state_editor_tutorial=self.generic_date,
+            last_started_state_translation_tutorial=self.generic_date,
+            last_logged_in=self.generic_date,
+            last_created_an_exploration=self.generic_date,
+            last_edited_an_exploration=self.generic_date,
+            profile_picture_data_url=self.generic_image_url,
+            default_dashboard='learner', creator_dashboard_display_pref='card',
+            user_bio=self.generic_user_bio,
+            subject_interests=self.generic_subject_interests,
+            first_contribution_msec=1,
+            preferred_language_codes=self.generic_language_codes,
+            preferred_site_language_code=(self.generic_language_codes[0]),
+            preferred_audio_language_code=(self.generic_language_codes[0])
+        ).put()
 
     def test_get_by_role(self):
         user = user_models.UserSettingsModel.get_by_role(
             feconf.ROLE_ID_ADMIN)
         self.assertEqual(user[0].role, feconf.ROLE_ID_ADMIN)
 
-
-class StoryProgressModelTests(test_utils.GenericTestBase):
-
-    def test_get_multi(self):
-        model = user_models.StoryProgressModel.create(
-            'user_id', 'story_id_1')
-        model.put()
-
-        model = user_models.StoryProgressModel.create(
-            'user_id', 'story_id_2')
-        model.put()
-
-        story_progress_models = user_models.StoryProgressModel.get_multi(
-            'user_id', ['story_id_1', 'story_id_2'])
-        self.assertEqual(len(story_progress_models), 2)
-        self.assertEqual(story_progress_models[0].user_id, 'user_id')
-        self.assertEqual(story_progress_models[0].story_id, 'story_id_1')
-
-        self.assertEqual(story_progress_models[1].user_id, 'user_id')
-        self.assertEqual(story_progress_models[1].story_id, 'story_id_2')
+    def test_export_data_trivial(self):
+        user = user_models.UserSettingsModel.get_by_role(
```

Comment:
Using get_by_role(feconf.ROLE_ID_ADMIN)[0] and [1] below is a bit unintuitive. Could you just get_by_id for the correct IDs?

Reply:
Done